api/backup_endpoints.go

```diff
@@ -1,11 +1,36 @@
 package api
 
 import (
-	"encoding/json"
 	"fmt"
 	"github.com/pkg/errors"
 )
 
+type BackupRemoteUploadResponse struct {
+	CompleteMultipartUpload string
+	AbortMultipartUpload    string
+	Parts                   []string
+	PartSize                int64
+}
+
+func (r *Request) GetBackupRemoteUploadURLs(backup string, size int64) (*BackupRemoteUploadResponse, error) {
+	resp, err := r.Get(fmt.Sprintf("/backups/%s?size=%d", backup, size), nil)
```

Comment:
You can do `D{"size": size}` in place of that second `nil` argument and it theoretically should attach it as a query parameter? At least that was the intention behind that argument.